mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -23,78 +37,148 @@ CLASS net/minecraft/class_2382 net/minecraft/util/math/Vec3i
 		ARG 1 vec
 	METHOD method_10260 getZ ()I
 	METHOD method_10262 getSquaredDistance (Lnet/minecraft/class_2382;)D
+		COMMENT {@return the squared distance between here and {@code vec}}
+		COMMENT
+		COMMENT @apiNote Passing {@link BlockPos} to this method will return a wrong value. Use
+		COMMENT {@link #getSquaredDistance(Vec3i, boolean)} with {@code treatAsBlockPos} set to
+		COMMENT {@code false} instead.
```

Comment:
Does it really return a wrong value? Most (if not all) usages of this method are passing `BlockPos`.

Reply:
`BlockPos.ORIGIN.getSquaredDistance(BlockPos.ORIGIN)` returns `0.75`.

Reply:
No, this means if the distance is from the center of the vec3i (like offset by 0.5 in all of x,y,z) to the given position as opposed to being on the exact grid point (edge of the block). Since we have `Vec3d.ofCenter()` which returns a `Vec3d` like this, I recommend renaming this parameter to `center` and document it.

Reply:
I'd say the method name is misleading then. I'm not sure if it's worth renaming to `getSquaredDistanceFromCenter` though.

Reply:
I'll document this - maybe rename, maybe not